.github/aur/flux-go/PKGBUILD.template

```diff
@@ -12,7 +12,7 @@ provides=("flux-bin")
 conflicts=("flux-bin")
 replaces=("flux-cli")
 depends=("glibc")
-makedepends=("go")
+makedepends=('go>=1.16', 'kustomize>=3.9', 'kustomize<4.0.0')
```

Comment:
@stefanprodan if versions lower than `3.9.x` work fine, please say so and I will make the range less strict.

Reply:
The manifests can be built with kustomize v4 since flux 0.9. The minimum supported version is 3.0 as we use `transformers`.

Reply:
Thank you, updated the build constraints.